guides/common/modules/snip_glossary-term-smart-proxy.adoc

```diff
@@ -0,0 +1,15 @@
+{SmartProxyServer}::
+ifdef::satellite[]
+{SmartProxyServer}s provide DHCP, DNS, and TFTP services and act as Ansible control node or Puppet server in separate networks.
+endif::[]
+ifndef::satellite[]
+{SmartProxyServer}s provide DHCP, DNS, and TFTP services and act as Ansible control node, Puppet server, or a Salt Master in separate networks.
+endif::[]
+They interact with {ProjectServer} in a client-server model.
```

Comment:
```suggestion
They mediate {Project} services between {ProjectServer} and hosts.
```
Would this be more precise/clearer?

Reply:
I don't think this conveys the same message as before.